.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/opt/anaconda3/envs/testing/bin/python"
```

Comment:
I'm guessing this a fix you needed to implement to make sure vscode was using the correct python path?